### ISSUE TITLE
Add contact for company / freelancers providing formation?

### ISSUE THREAD

**Issue by DamnClin** [jhipster/jhipster-lite#6808]
We talked about that like years ago, do we add a formation / consultancy section in the readme with contacts to trusted members / companies?

@pascalgrimaud your advice on that?

**Comment by pascalgrimaud** [comment]
Yes, it's related to this ticket: https://github.com/jhipster/generator-jhipster/issues/21076

After discussing with some members, I'd prefer to add name of trusted members, instead of company. As it's too easy to say 'compagny supports jhipster and it's not really active'.

As you're the lead of JHipster Lite, we can put your name there. I'll be happy with that.

**Comment by DamnClin** [comment]
I was thinking about something for JHLite, I'm not certain I'm suited to provide formation for generator :D

**Comment by pascalgrimaud** [comment]
Yes sure, but the best place to have visibility is the official web site, and we can mention jhlite for you

**Comment by renanfranca** [comment]
I am interested in this. Is it possible to put my name as well?

Thanks 😁
